.travis.yml

```diff
@@ -6,7 +6,8 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
-  - 2.3.1
+  - 2.3.3
+  - ruby-head
```

Comment:
`ruby-head` is under development that is not stable. I think that add `ruby-head` to `allow_failures`.

cf. https://github.com/octokit/octokit.rb/blob/3887fd52134f56032b871304e56bd0f1753b0dc5/.travis.yml#L27-L30